lld/test/ELF/linkerscript/enable-non-contiguous-regions.test

```diff
@@ -0,0 +1,279 @@
+# REQUIRES: x86
+
+# RUN: split-file %s %t
```

Comment:
`RUN: rm -rf %t && split-file %s %t && cd %t`

The convention is to remove %t to avoid prevent lingering files. `cd %t` allows to remove `%t/` below, making RUN lines cleaner.